src/DefinitionResolver.php

```diff
@@ -716,6 +603,7 @@ public function resolveExpressionNodeToType($expr)
             if ($token === PhpParser\TokenKind::NullReservedWord) {
                 return new Types\Null_;
             }
+            return new Types\Mixed_;
```

Comment:
Do you expect this case to ever get hit? Or is it for future-proofing?

Reply:
These were added accidentally (same goes the `Types\Mixed_` below) - I tried various things in the same branch if they improved performance, including bailing early here. It did not really affect performance, but I apparently forgot to revert some changes. I can revert this.